packages/core/src/DateTimePicker/DateTimePicker.tsx

```diff
@@ -121,6 +121,11 @@ export interface DateTimePickerProps extends ModalProps {
    * Default `AM` and `PM`
    */
   readonly hour12MeridiemLabels?: TranslationMeridiem
+  /**
+   * If `true`, clicking outside the DateTimePicker is possible,
+   * `false` does not allow clicking outside the modal.
+   */
+  readonly allowOutsideClick?: boolean
```

Comment:
`DateTimePickerProps` extends `ModalProps` so you don't need it here.

Reply:
Fixed.